test/Microsoft.CodeAnalysis.Razor.Workspaces.Test/ProjectSystem/DefaultProjectSnapshotManagerTest.cs

```diff
@@ -342,13 +328,12 @@ public void WorkspaceProjectChanged_BackgroundUpdate_StillDirty_WithSignificantC
             // Assert
             snapshot = ProjectManager.GetSnapshot(project);
             Assert.True(snapshot.IsDirty);
-            Assert.Same(configuration, snapshot.Configuration);
 
             Assert.True(ProjectManager.ListenersNotified);
             Assert.True(ProjectManager.WorkerStarted);
         }
 
-        [Fact]
+        [Fact(Skip = "We no longer have any background-computed state")]
```

Comment:
I had to skip two tests here because they test the case where a background update has occurred but didn't result in a state change. Since we don't have any background updates (yet) we don't have the facility to test this scenario.